site/gatsby-site/page-creators/createCitationPages.js

```diff
@@ -188,6 +192,62 @@ const createCitationPages = async (graphql, createPage) => {
     ...allMongodbAiidprodResources.nodes.map((r) => ({ ...r, namespace: 'resources' })),
   ];
 
+  let spacialIncidents;
+
+  const nlpStateResponse = await axios.get(
+    'https://raw.githubusercontent.com/responsible-ai-collaborative/nlp-lambdas/main/inference/db_state/state.csv'
+  );
+
+  if (nlpStateResponse?.status == 200) {
+    const lines = nlpStateResponse.data
+      .split('\n')
+      .slice(1)
+      .filter((line) => line.length > 2);
+
+    const embeddings = lines.map((line) => JSON.parse(line.split('"')[1]));
+
+    const ids = lines.map((line) => line.split(',')[0]);
+
+    const model = new TSNE({
+      dim: 2,
```

Comment:
How much have you played with these settings?

Reply:
Originally I just copied the ones used in the tsne-js readme. I spent a couple hours today playing with the parameters, and for the most part changing them seemed to produce subjectively worse results or no noticeable difference. The exceptions was the number of iterations, which I increased by a lot since this only runs at build time. I also lowered the `earlyExaggeration` a little bit, which feels like it maybe helped but I'm not confident about.